### PR TITLE
Train booking website

### PR DESCRIPTION
**Pull request by pulkit1417**
### Issue Title: Train Booking Website

- **Name of the project** : Train Booking Website
- **Category:** Angular-JS-Projects
- **Name:** Pulkit Gupta
- **GitHub ID:** https://github.com/pulkit1417
- **Idenitfy yourself:**  GSSOC Contributor

Closes: #169 

### Describe the add-ons or changes you've made 📃

Made a whole new Train Booking Website which helps you in book tarin when you want to travel.

### How Has This Been Tested? ⚙️

This has been tested by making users book trains for their travels.

### Checklist: ☑️

- [x] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CONTRIBUTING.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CODE_OF_CONDUCT.md) of this project.
- [x] This PR does not contain plagiarized content.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] My changes generate no new warnings.

### Screenshots 📷
screenshot webp

### Note to reviewers 📄
I would love to know about the opinions of the reviewers so that if there are any changes to be made that can be done easily, moreover we can connect for collaborating for similar types of projects

### CODE REVIEW

File: Angular-JS-Projects/Intermediate/Train-Website/package-lock.json
Comment (Avdhesh-Varshney) [inline Angular-JS-Projects/Intermediate/Train-Website/package-lock.json:1]: delete this file not required.

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]
@pulkit1417 enlist your project in the readme file of `Angular-JS-Projects` and add it to the database

**Comment by pulkit1417** [comment]
Hello, @Avdhesh-Varshney I have made the requested changes by you please review it once again and please let me know if there are any further changes to be made.

**Review by Avdhesh-Varshney — APPROVED** [review]
@pulkit1417 PR Approved 🎉

**Pull request merged**